pkg/drivers/hyperkit/driver_test.go

```diff
@@ -72,15 +73,5 @@ func testEq(a, b []int) bool {
 		return false
 	}
 
-	if len(a) != len(b) {
-		return false
-	}
-
-	for i := range a {
-		if a[i] != b[i] {
-			return false
-		}
-	}
-
-	return true
+	return slices.Equal(a, b)
```

Comment:
We can drop testEqual and use slices.Equal directly. But this driver is deprecated so it is not worth the time to improve the tests. We should focus on vfkit and krunkit drivers for macOS.